packages/node-experimental/src/sdk/api.ts

```diff
@@ -0,0 +1,225 @@
+import { GLOBAL_OBJ, consoleSandbox, dateTimestampInSeconds } from '@sentry/utils';
+import { getScopesFromContext, setScopesOnContext } from '../utils/contextData';
+
+import type { ExclusiveEventHintOrCaptureContext } from '../utils/prepareEvent';
+import { parseEventHintOrCaptureContext } from '../utils/prepareEvent';
+import type { Scope } from './scope';
+import { getClient, getCurrentScope, getGlobalScope, getIsolationScope } from './scope';
+
+export { getCurrentScope, getGlobalScope, getIsolationScope, getClient };
+export { setCurrentScope, setIsolationScope } from './scope';
+
+/**
+ * Fork a scope from the current scope, and make it the current scope in the given callback
+ */
+export function withScope<T>(callback: (scope: Scope) => T): T {
+  return context.with(context.active(), () => callback(getCurrentScope()));
+}
+
+/**
+ * Fork a scope from the current scope, and make it the current scope in the given callback.
+ * Additionally, also setup a new isolation scope for the callback.
+ */
+export function withIsolationScope<T>(callback: (scope: Scope, isolationScope: Scope) => T): T {
+  const ctx = context.active();
+  const currentScopes = getScopesFromContext(ctx);
+  const scopes = currentScopes
+    ? { ...currentScopes }
+    : {
+        scope: getCurrentScope(),
+        isolationScope: getIsolationScope(),
+      };
+
+  scopes.isolationScope = scopes.isolationScope.clone();
+
+  return context.with(setScopesOnContext(ctx, scopes), () => {
+    return callback(getCurrentScope(), getIsolationScope());
+  });
+}
+
+/** Get the ID of the last sent error event. */
+export function lastEventId(): string | undefined {
+  return getCurrentScope().lastEventId();
+}
+
+/**
+ * Configure teh current scope.
+ * @deprecated Use `getCurrentScope()` instead.
+ */
+export function configureScope(callback: (scope: Scope) => void): void {
+  callback(getCurrentScope());
+}
+
+/** Capture an exception to Sentry. */
+export function captureException(exception: unknown, hint?: ExclusiveEventHintOrCaptureContext): string {
+  return getCurrentScope().captureException(exception, parseEventHintOrCaptureContext(hint));
+}
+
+/** Capture a message to Sentry. */
+export function captureMessage(
+  message: string,
+  // eslint-disable-next-line deprecation/deprecation
+  captureContext?: CaptureContext | Severity | SeverityLevel,
+): string {
+  // This is necessary to provide explicit scopes upgrade, without changing the original
+  // arity of the `captureMessage(message, level)` method.
+  const level = typeof captureContext === 'string' ? captureContext : undefined;
+  const context = typeof captureContext !== 'string' ? { captureContext } : undefined;
+
+  return getCurrentScope().captureMessage(message, level, context);
+}
+
+/** Capture a generic event to Sentry. */
+export function captureEvent(event: Event, hint?: EventHint): string {
+  return getCurrentScope().captureEvent(event, hint);
+}
+
+/**
+ * Add a breadcrumb.
```

Comment:
```suggestion
 * Add a breadcrumb to the current isolation scope.
```

(for consistency with the other functions?)